### PR TITLE
Addition of the parameter 'how to handle replicates' to all tools

### PR DESCRIPTION
**Pull request by Delphine-L**
FOR CONTRIBUTOR:
* [x] - I have read the [CONTRIBUTING.md](https://github.com/galaxyproject/tools-iuc/blob/master/CONTRIBUTING.md) document and this tool is appropriate for the tools-iuc repo.
* [ ] - License permits unrestricted use (educational + commercial)
* [ ] - This PR adds a new tool or tool collection
* [x] - This PR updates an existing tool or tool collection
* [ ] - This PR does something else (explain below)

### CODE REVIEW

File: tools/transit/macros.xml [hunk 1]
```diff
@@ -18,10 +18,10 @@
         <yield />
         <data name="sites" from_work_dir="transit_out.txt" format="tabular" label="${tool.name} on ${on_string} Sites" />
     </xml>
-    <xml name="replicates">
-        <param name="replicates" type="select" label="How to handle replicates">
-        	<option value="Mean">Mean</option>
+    <xml name="hand_replicates">
```
Comment (bgruening) [inline tools/transit/macros.xml]: can we name this `handle_replicates`?

File: tools/transit/transit_hmm.xml [hunk 2]
```diff
@@ -62,7 +61,7 @@ Note : Intended only for Himar1 datasets.
 
 
 Optional Arguments:
-|    -r <string>     :=  How to handle replicates. Sum, Mean. Default: -r Mean
+|    -r <string>     :=  How to handle replicates. Sum, ß. Default: -r Mean
```
Comment (bgruening) [inline tools/transit/transit_hmm.xml]: above you also call it mean, why here ß?

Comment (Delphine-L) [reply tools/transit/transit_hmm.xml]: That was a typo, thank you for seing it !

**Comment by bgruening** [comment]
Merci!

**Pull request merged**